lib/mixins/generate-id.js

```diff
@@ -4,12 +4,7 @@
  * it is better to prefer references/refs and events over relying on generated id.
  * @see https://github.com/bootstrap-vue/bootstrap-vue/issues/281
  *
- * Currently is being used in:
- *  - dropdown.vue
- *  - form-checkbox-vue
- *  - form-file.vue
- *  - form-input.vue
- *  - form-select.vue
+ * No longer used
```

Comment:
I think it is.  I went through most of the components to see if it was referenced anywhere else and didn't s it.